From: Stefan Bader <stefan.bader@canonical.com>
Date: Wed, 27 May 2015 14:57:05 +0200
Subject: ubuntu: libxl: Set common qemu path

Upstream Xen assumes that qemu-system-x86 is shipped as part of the Xen
package. However, since this is now merged back into common qemu and
we (and Debian) do not ship an additional binary, the binary is not
found under the LIBXL_EXECBIN_DIR. So instead, refer to the common
path (/usr/bin).

*Update Artful*
This change is still needed but was split from the qemu-dm compat bits
that were in the same patch before. Those changes are logically a different
change and could be dropped as that old compat is no more needed.

BugLink: http://bugs.launchpad.net/bugs/1459600

Signed-off-by: Stefan Bader <stefan.bader@canonical.com>

--- a/src/libxl/libxl_capabilities.c
+++ b/src/libxl/libxl_capabilities.c
@@ -506,7 +506,7 @@
         if ((guest = virCapabilitiesAddGuest(caps,
                                              guest_archs[i].hvm ? VIR_DOMAIN_OSTYPE_HVM : VIR_DOMAIN_OSTYPE_XEN,
                                              guest_archs[i].arch,
-                                             LIBXL_EXECBIN_DIR "/qemu-system-i386",
+                                             "/usr/bin/qemu-system-i386",
                                              (guest_archs[i].hvm ?
                                               LIBXL_FIRMWARE_DIR "/hvmloader" :
                                               NULL),
